eth_pydantic_types/hashing.py

```diff
@@ -0,0 +1,13 @@
+from typing import Tuple
```

Comment:
idea: maybe we should rename `validators.py` to like `utils.py` and include these new functions there